### ISSUE TITLE
Include rankmirrors

### ISSUE THREAD

**Issue by mrcasquinhas** [MatMoul/archfi#24]
Amazing work.
But I've noticed that updating is too slow, so I did rankmirrors from https://wiki.archlinux.org/index.php/mirrors
So, maybe it should be better to rankmirrors before updating, after you edit the mirrors. But do it even of you don't edit mirror list.
Thank you.

**Comment by MatMoul** [comment]
Thanks for return, I'll try to provide an option for this.

**Comment by MatMoul** [comment]
I'm not sure this feature are usefull...
If you use the last Archlinux media, rankmirror isn't needed...

**Comment by retro-wertz** [comment]
+1 for rankmirrors, or reflector and allow generating mirrors based on location for faster downloads sorted by latest sync for faster updates
thanks.

**Comment by workdowg** [comment]
I also agree. I have other Arch systems so I just copied it via wget from my server but others trying out Arch don't have this advantage.

**Comment by ejbolt** [comment]
I can see this being useful, but is rankmirrors not included in the iso?  I
can't remember if I read if it stopped being included by default, in which
case shame on the Arch maintainers of the isos as this is incredibly
important to include.  I have mixed feelings on whether this should be
implemented, as it's something rather simple for all of us to do, and this
script should really be used after someone has installed Arch manually so
they know the ins and outs   Perhaps I'm in the minority, and perhaps
that's isn't MatMoul's intent, in which case I apologise for being wrong,
but that's just my opinion on this :).

**Comment by retro-wertz** [comment]
ok thats too bad.

**Comment by MatMoul** [comment]
@retro-wertz Sorry but it's too difficult to manage the small space available when you boot with the arch iso...
And installing additional packages need space
